src/main/scala/Vingt_Et_Un.scala

```diff
@@ -0,0 +1,72 @@
+object Vingt_Et_Un {
+
+  val Jack = "J"
+  val Queen = "Q"
+  val King = "K"
+  val Ace = "A"
+
+
+  def parse(card: String): Int = {
+    val cardAsInteger = toInt(card)
+    if(cardAsInteger >= 2 && cardAsInteger <=10 ){
+      cardAsInteger
+    }else{
+      card match{
+        case Jack | Queen | King => 10
```

Comment:
A bit devious else statement + pattern match. 

In the case of ```card = "24"``` for example you will get a `0`(due to your `toInt` function), go to the else statement and then get a match exception, which you haven't denoted explicitly anywhere. 
you could solve this by doing for example
```scala
card match{
  case ....
  case _ => throw new IllegalArgumentException("Not a card")
  }
```
Which is the lesser evil, the best way is with `Option`, which we cover in this and next week's lectures.

Generally it's also not so nice combining if/else and match